python/lsst/ts/m2gui/controltab/tab_overview.py

```diff
@@ -71,7 +71,8 @@ def __init__(self, title, model):
         # Internal layout
         self._layout = self._set_layout()
```

Comment:
You don't need self._layout - you don't need to store reference to layout you are using (adding it to widget with setLayout). With that, you can shorten all that to only:
```
self.widget().setLayout(self._set_layout())
```
with the same caveats apply to _set_layout name - it doesn't set anything, so name it create_layout? Or even better there isn't any harm to create & set layout in constructor,. most of PySide2 programs do that. So dropping _set_layout at all and move it to __init__? You really don't need to assume you will set a different layout, this is used only sometime and shall not be a rule.

Reply:
Fixed in the commit: [a26aa99](https://github.com/lsst-ts/ts_m2gui/pull/5/commits/a26aa99a148bac5d0f2ecb765ca284af2d2f51dd). Thanks!